io-engine/src/subsys/config/opts.rs

```diff
@@ -525,7 +527,8 @@ impl From<&BdevOpts> for spdk_bdev_opts {
             bdev_auto_examine: false,
             reserved9: Default::default(),
             opts_size: std::mem::size_of::<spdk_bdev_opts>() as u64,
-            reserved: Default::default(),
+            iobuf_small_cache_size: Default::default(),
```

Comment:
Are these something we may want to set to specific value through env variable? Similar to the `bdev_io_*_size` ?

Reply:
Fixed